swan-lake/learn/calling-java-code-from-ballerina.md

```diff
@@ -84,17 +84,18 @@ This section assumes that you have already read [How to Structure Ballerina Code
 
 #### Creating a Ballerina Project
 ```sh
-> ballerina new yaml-project
-Created new Ballerina project at yaml-project
+> ballerina new yaml_project
```

Comment:
We should replace all occurrences of project to package

Reply:
Done